### ISSUE TITLE
Can not use my own ML program

### ISSUE THREAD

**Issue by zl8255** [HewlettPackard/swarm-learning#66]
Dear RadhakrishnaJ,

I write my own ML program according to the illustration in “Adapting ML programs for Swarm Learning”. And I have replaced the model file from "swarm-learning/examples/mnist-pytorch/model/mnist_py.py", but while execute the example, it has some errors, can you give me some methods to solve it?

The error happends in "swarm.py" in line 393

![image](https://user-images.githubusercontent.com/50286580/142593460-bca976d6-f33d-4d64-9ec9-395f91fe0170.png)

Thank You and Best Regard.

**Comment by zl8255** [comment]
I have test the ML program and just add the Swarm learning callback

**Comment by RadhakrishnaJ** [comment]
Hello, 

Make sure your ML model is running in a standalone mode first without swarm callback first. Once it is trains locally then add Swarmcallback.

Please confirm this. If you still feel issue is with Swarmcallback then please fill in issue template for further support.

**Comment by zl8255** [comment]
Thanks for you fast reply @RadhakrishnaJ

I have run my ML model in a standalone mode locally and there are no errors. Then I just add Swarmcallback without any other changing. Could you give me some suggestions to solve it?

**Comment by zl8255** [comment]
I upload my test model file here, can you have a test in your convenience？  
[test_model.zip](https://github.com/HewlettPackard/swarm-learning/files/7570669/test_model.zip)

Thank You and Best Regard.

**Comment by zl8255** [comment]
I have solved this problem because "nn.BatchNorm2d" is used in the neural network I created, although I don't know why the reason is. Because it can be executed normally  in a standalone mode.  There may be some incompatibilities or my reason.Thanks again.

**Comment by zl8255** [comment]
done

**Comment by RadhakrishnaJ** [comment]
Thanks for the update. Closing this issue now.
